client/app/telecom/telephony/line/phone/order/telecom-telephony-line-phone-order.html

```diff
@@ -245,35 +245,44 @@ <h3 data-translate="telephony_line_phone_order_shipping_contact"></h3>
 
                     <!-- PRICE SUMMARY TABLE -->
                     <h3 data-translate="telephony_line_phone_order_summary"></h3>
-                    <div class="responsive-table-line">
-                        <table class="table table-condensed">
+                    <div class="table-responsive">
+                        <table class="table"
+                               data-ng-class="{
+                                   'table-hover table-striped': PhoneOrderCtrl.order.summary.details.length > 1
+                               }">
                             <thead>
                                 <tr>
-                                    <th data-translate="telephony_line_phone_order_table_description"></th>
-                                    <th class="text-sm-right"
+                                    <th scope="col"
+                                        data-translate="telephony_line_phone_order_table_description">
+                                    </th>
+                                    <th scope="col"
+                                        class="text-sm-right"
```

Comment:
Does `text-sm-right` & `text-right-sm` mean the same thing? You're using both in this PR